semeio/workflows/correlated_observations_scaling/scaled_matrix.py

```diff
@@ -61,9 +57,6 @@ def get_scaling_factor(self, events):
                 below a user threshold
         """
         data_matrix = self.get_data_matrix()
-        nr_components, singular_values = self.get_nr_primary_components(
-            threshold=events.threshold
-        )
         nr_observations = data_matrix.shape[1]
 
         logging.info("Scaling factor calculated from {}".format(events.keys))
```

Comment:
It is a bit weird that the only reason for including `events` in the function is to add this very general log statement. Suppose it could've been moved to outside of the function call and rather logged on the other side?

Reply:
You raise a valid point, a bit besides the point of the issue, but I will fix it.